package.json

```diff
@@ -88,7 +93,14 @@
     "stylelint-scss": "3.5.4",
     "stylelint-webpack-plugin": "0.10.5",
     "supertest": "3.4.2",
+    "ts-loader": "5.4.5",
```

Comment:
`ts-loader` is also not need it. We use `babel7` for transpile Typescript to ES5. See here an example.

https://github.com/verdaccio/verdaccio-htpasswd/blob/master/.babelrc

```
{
  "presets": [["@verdaccio", {"typescript": true}]]
}
```